src/index.html

```diff
@@ -11,7 +11,169 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" class="form">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form__fields" for="surname">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label class="form__fields" for="name">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label class="form__fields" for="age">
+          How old are You?
+          <input 
+          type="number"
+            min="1"
+            max="100"
+            value="12"
+            name="age"
+            id="age"
+          >
+        </label>
+        <label class="form__fields" for="dateOfBirth">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            id="dateOfBirth"
+          >
+        </label>
+        <label class="form__fields" for="accept">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+            id="accept"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form__fields" for="email">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required autocomplete="off"
+          >
+        </label>
+        <label class="form__fields" for="password">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+            minlength="7"
+            maxlength="33"
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form__fields">
```

Comment:
leave labels only for inputs here
it chooses the first options, when you click on Do you love cats?